docs/basics/connection-approval.md

```diff
@@ -61,13 +62,64 @@ private void ApprovalCheck(NetworkManager.ConnectionApprovalRequest request, Net
 
     // Rotation to spawn the player object (if null it uses the default of Quaternion.identity)
     response.Rotation = Quaternion.identity;
+    
+    // If response.Approved is false, you can provide a message that explains the reason why via ConnectionApprovalResponse.Reason
+    // On the client-side, NetworkManager.DisconnectReason will be populated with this message via DisconnectReasonMessage
+    response.Reason = "Some reason for not approving the client";
 
     // If additional approval steps are needed, set this to true until the additional steps are complete
     // once it transitions from true to false the connection approval response will be processed.
     response.Pending = false;
 }
 ```
 
+## Sending An Approval Declined Reason (_`NetworkManager.ConnectionApprovalResponse.Reason`_)
+
+Under the condition that you need to deny a player from connecting for any particular reason (*reached maximum number of connections, invalid authorization, etc.*), the `NetworkManager.ConnectionApprovalResponse` structure provides you with the optional `NetworkManager.ConnectionApprovalResponse.Reason` property.  When `NetworkManager.ConnectionApprovalResponse.Approved` is false and `NetworkManager.ConnectionApprovalResponse.Reason` has been populated with the reason for denying the player's request to connect, then the server will send the client a `DisconnectReasonMessage`. Upon the client side receiving the `DisconnectReasonMessage`, the `NetworkManager.DisconnectReason` property will be populated with the `NetworkManager.ConnectionApprovalResponse.Reason` message. The example provided below demonstrates how this works:
+
```

Comment:
If you need to deny a player from connecting for any reason (such as reaching the maximum number of connections or an invalid authorization), you can use the optional `NetworkManager.ConnectionApprovalResponse.Reason` property on the `NetworkManager.ConnectionApprovalResponse` structure. When `NetworkManager.ConnectionApprovalResponse.Approved` is false and `NetworkManager.ConnectionApprovalResponse.Reason` has been populated with the reason for denying the player's request to connect, the server sends the client a `DisconnectReasonMessage`. When the client receives the `DisconnectReasonMessage`, the `NetworkManager.DisconnectReason` property populates with the `NetworkManager.ConnectionApprovalResponse.Reason` message. The example below shows how this works: